src/services/device.js

```diff
@@ -46,11 +46,14 @@ function getHistory(readings) {
   };
 }
 
-export function getReadings(deviceUID) {
+export function getReadings(deviceUID, timeframe) {
   return fetch(
-    `${SERVER_URL}/?` +
+    // Temporarily hit the server on the PR
+    "https://airnote-api-pr-45.onrender.com/?" +
```

Comment:
Obviously we can't merge with the hard-coded URL. Is it possible to update the `SERVER_URL` environment variable instead?

Reply:
This hardcoded version was added just for a "staging" version of the app that Michelle (or a PR reviewer) see without having to spin the app up locally.

TJ asked the [Render team](https://community.render.com/t/static-site-and-web-service-communication-in-pull-requests/5218/2) about this and they replied back, so I will try to implement their solution for the future.